### PR TITLE
Aden 1467 instant globals update

### PR DESCRIPTION
**Pull request by bkoval**
@gabrys  @naumovs This should introduce two changes as stated in https://wikia-inc.atlassian.net/browse/ADEN-1556:

1) Possibility of overriding boolean InstantGlobals by appending InstantGlobals.=[0|1] to the querystring
2) Bringing InstantGlobals to WikiaMobile

### CODE REVIEW

File: extensions/wikia/AdEngine/js/AdContext.js [hunk 1]
```diff
@@ -3,7 +3,11 @@
  * The AMD module to hold all the context needed for the client-side scripts to run.
  */
 define('ext.wikia.adEngine.adContext', [
-	'wikia.window', 'wikia.document', 'wikia.geo', require.optional('wikia.instantGlobals'),  require.optional('wikia.abTest')
+	'wikia.window',
+	'wikia.document',
+	'wikia.geo',
+	'wikia.instantGlobals',
```
Comment (bkoval) [inline extensions/wikia/AdEngine/js/AdContext.js]: Let's see if it doesn't fail on Mercury.

File: extensions/wikia/AssetsManager/config.php [hunk 2]
```diff
@@ -611,6 +611,7 @@
 		'//resources/wikia/libraries/mustache/mustache.js',
 		'//resources/wikia/libraries/sloth/sloth.js',
 		'//resources/wikia/modules/thumbnailer.js',
+		'//resources/wikia/modules/instantGlobals.js',
```
Comment (bkoval) [inline extensions/wikia/AssetsManager/config.php]: Let's consider using package that is common for wikiamobile and mercury.

File: extensions/wikia/AdEngine/js/AdContext.js (hunk 1 above)
Comment (bkoval) [inline extensions/wikia/AdEngine/js/AdContext.js]: Seems like it's not failing after moving js module to the package common for both mercury and wikiamobile.

Comment (gabrys) [inline extensions/wikia/AdEngine/js/AdContext.js]: (y)

**Comment by gabrys** [comment]
LATM

**Pull request merged**